test/jdk/java/net/vthread/HttpALot.java

```diff
@@ -85,11 +95,15 @@ public static void main(String[] args) throws Exception {
 
         // go
         server.start();
-        try {
+        try (serverExecutor) {
```

Comment:
This part looks good, as does checking that the task result is "Hello" as that adds more checking.

Reply:
Done. I've updated the PR to close the executor after the server is stopped.